packages/wallets/src/components/WalletCurrencyIcon/currencies.ts

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const currencies = {
+    aud: React.lazy(() => import('../../public/images/currencies/aud.svg')),
+    btc: React.lazy(() => import('../../public/images/currencies/btc.svg')),
+    eth: React.lazy(() => import('../../public/images/currencies/eth.svg')),
+    eur: React.lazy(() => import('../../public/images/currencies/eur.svg')),
+    eusdt: React.lazy(() => import('../../public/images/currencies/eusdt.svg')),
+    ltc: React.lazy(() => import('../../public/images/currencies/ltc.svg')),
+    usd: React.lazy(() => import('../../public/images/currencies/usd.svg')),
+    gbp: React.lazy(() => import('../../public/images/currencies/gbp.svg')),
+    usdc: React.lazy(() => import('../../public/images/currencies/usdc.svg')),
+    tusdt: React.lazy(() => import('../../public/images/currencies/eusdt.svg')),
+    ust: React.lazy(() => import('../../public/images/currencies/eusdt.svg')),
+};
```

Comment:
Lets not lazy load icons 😅   we should do it by fold

Reply:
yeaa i shouldn't have done this as well, just realized we are importing all of these anyways on the wallets page

Reply:
heheh ok

Reply:
fix ya

Reply:
yes senpaii